.ci/bump-cs-version-in-esclipsecs-sevntu-plugin.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+
+if [[ -z $1 ]]; then
+  echo "ERROR: version is not set"
+  echo "Usage: bump-cs-version-in-eclipsecs-sevntu-plugin.sh <version>"
+  exit 1
+fi
+
+VERSION=$1
+cd eclipsecs-sevntu-plugin
+
+mvn versions:set-property -Dproperty=checkstyle.version -DnewVersion="$VERSION"
+rm pom.xml.versionsBackup
```

Comment:
Is there options to do update without backup files, to not create them by maven at all

Reply:
https://stackoverflow.com/a/15978219/1015848

Reply:
Shall I apply to  `bump-cs-version-in-sonar-plugin.sh` and `bump-cs-version-in-sevntu-checks.sh` as well?

Reply:
If not complicated, please do

Reply:
Done.